app/admin/pages.rb

```diff
@@ -107,20 +111,24 @@
             para component&.subtopic_description if pc.component_type == 'PageHeader2Component'
             # Alignment
             para "Alignment: #{component&.alignment}" if pc.component_type == 'PageHeader3Component'
+
+            para component&.text.html_safe unless pc.component_type == 'PageHrComponent' || pc.component_type == 'PagePracticeListComponent' || pc.component_type == 'PageHeader2Component' || pc.component_type == 'PageSubpageHyperlinkComponent' || pc.component_type == 'PageHeader3Component' || pc.component_type == 'PageYouTubePlayerComponent' || pc.component_type == 'PageImageComponent' || pc.component_type == 'PageDownloadableFileComponent' || pc.component_type == 'PageCtaComponent' || pc.component_type == 'PageMapComponent'
+
```

Comment:
Can we change this back to the format that is in the feature branch? AKA:
```
# Text
if (pc.component_type == 'PageAccordionComponent' ||
    pc.component_type == 'PageParagraphComponent' ||
    pc.component_type == 'PageCompoundBodyComponent') && component&.text.present?
       para component.text.html_safe
end
```

Reply:
Done

Reply:
You resolved this, but didn't change it back

Can we change this back to the format that is in the feature branch? AKA:
```
# Text
if (pc.component_type == 'PageAccordionComponent' ||
    pc.component_type == 'PageParagraphComponent' ||
    pc.component_type == 'PageCompoundBodyComponent') && component&.text.present?
       para component.text.html_safe
end
```

Reply:
Set it back to the if statement vs unless ^^